src/_P035_IRTX.ino

```diff
@@ -218,14 +218,14 @@ boolean Plugin_035(byte function, struct EventStruct *event, String& string)
                        if (i+2 >= total || idx + (rep = from_32hex(IrRaw[i++])) * 2 > sizeof(buf)){
                          delete[] buf;
                          buf = nullptr;
-                         return addErrorTrue("Invalid RAW2 B32 encoding!");
+                         return addErrorTrue(F("Invalid RAW2 B32 encoding!"));
```

Comment:
Hmm now I know why these were not using the F-macro. They cause build errors :(

Reply:
@TD-er I can fix it in next commit

Reply:
Already done and merged ;)